### PR TITLE
Refactor: multiple verifiers in language files

### LINKED ISSUE #338: Refactor language files for multiple verifiers
## Background

We need message strings for multiple verifiers

## Tasks

* [x] Create a new [verifier fixture](https://github.com/cal-itp/benefits/blob/dev/fixtures/02_eligibilityverifier.json) from the existing one
  * [x] With a new `pk` and `fields.name`
  * [x] Adjust both to have different `msgid`, e.g. `eligibility.pages.confirm.dmv.*` and `eligibility.pages.confirm.mst.*`
* [x] Adjust the `django.po` files with new `msgid` and new sample content for the Courtesy Cards verifier for each language
* [x] Wire the above fixture to one of the [sample agencies](https://github.com/cal-itp/benefits/blob/dev/fixtures/04_transitagency.json)

## Testing

Once #322 is merged

* [x] Remove your local db and local compiled language files, then run `bin/init.sh`
  * [x] Observe that the migrations run correctly, the fixture data is loaded, and the language files are compiled
* [x] Open the app locally and select the agency configured above with multiple verifiers
  * [x] Observe the `eligibility:index` view with different verifier selections
* [x] Select one of the verifiers
  * [x] Observe the verifier-specific `eligibility:start` view
  * [x] Confirm verification of a test user
* [x] Go back to `eligibility:index` and select the other verifier
  * [x] Observe the verifier-specific `eligibility:start` view
  * [x] Confirm verification of a test user

### PR DESCRIPTION
**Pull request by thekaveman**
Closes #338 

✅ Can be rebased on `dev` once #337 is merged to incorporate those changes as well.

### CODE REVIEW

File: benefits/eligibility/forms.py [hunk 1]
```diff
@@ -20,11 +20,11 @@ class EligibilityVerifierSelectionForm(forms.Form):
 
     verifier = forms.ChoiceField(label="", widget=forms.RadioSelect)
 
-    submit_value = _("eligibility.pages.index.button")
+    submit_value = _("eligibility.buttons.continue")
 
     def __init__(self, agency: models.TransitAgency, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        choices = [(v.id, v.selection_label) for v in agency.eligibility_verifiers.all()]
+        choices = [(v.id, _(v.selection_label)) for v in agency.eligibility_verifiers.all()]
```
Comment (angela-tran) [inline benefits/eligibility/forms.py:27]: Nice catch 🙏

File: benefits/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -433,3 +359,90 @@ msgid "enrollment.pages.success.p2"
...
+msgstr "Your MST Courtesy Card"
+
+msgid "eligibility.pages.start.mst.items[0].text"
+msgstr "An active card that has not expired"
+
+msgid "eligibility.pages.start.mst.p[0]"
+msgstr ""
+"This program is currently open to all MST Courtesy Cardholders. Not a "
+"Courtesy Cardholder? Get in touch with your transit provider to learn "
+"about other available discount programs at"
```
Comment (angela-tran) [inline benefits/locale/en/LC_MESSAGES/django.po:425]: **question**: Will a separate issue handle adding the transit agency phone number after this string?

**Review by angela-tran — COMMENTED** [review]
That's a bummer we can't have multiple files for a language. The msgids you came up with for each verifier look good though! 💥 

I just have the one question about `eligibility.pages.start.mst.p[0]`.

I also noticed bug while following the "Testing" steps in #338, but it's unrelated to these changes, so I created issue #346.

File: benefits/locale/en/LC_MESSAGES/django.po (hunk 2 above)
Comment (thekaveman) [reply benefits/locale/en/LC_MESSAGES/django.po:425]: I created #349

**Review by angela-tran — APPROVED** [review]

**Pull request merged**